packages/block-editor/src/components/convert-to-group-buttons/use-convert-to-group-button-props.js

```diff
@@ -31,68 +31,62 @@ import { store as blockEditorStore } from '../../store';
  * @return {ConvertToGroupButtonProps} Returns the properties needed by `ConvertToGroupButton`.
  */
 export default function useConvertToGroupButtonProps( selectedClientIds ) {
-	const {
-		clientIds,
-		isGroupable,
-		isUngroupable,
-		blocksSelection,
-		groupingBlockName,
-	} = useSelect(
+	return useSelect(
```

Comment:
In 9a2fb4a7ec607599b633424ee95ac3e7fe32980f I took the liberty of refactoring this hook, which was IMO adding unnecessary noise and was repacking objects for no reason:

```js
const { a, b, c } = useSelect( ( select ) => {
  const _a = ...
  const _b = ...
  const _c = ...
  return { a: _a, b: _b, c: _c }
} )
return { a, b, c }
```

becomes

```js
return useSelect( ( select ) => {
  const a = ...
  const b = ...
  const c = ...
  return { a, b, c }
} )
```

Reply:
I didn't notice that. That's better, thank you!